extensions/sonarqube/overview.md

```diff
@@ -1,11 +1,9 @@
 **[SonarQube][sq]** is the leading product for Continuous Code Quality. It supports all major programming languages, including C#, VB .Net, JavaScript, TypeScript, C/C++ and many more. Integrate it in your on-premise TFS installation, and continuously track down bugs and vulnerabilities in your codebase.
 
-If you are new to SonarQube, using VSTS or looking for cloud-based solutions, then make sure to also check our dedicated [SonarCloud extension for VSTS][sc_vsts] !
-
 This TFS extension provides build tasks that you can add in your build definition. All branches and pull-requests are automatically analyzed, allowing you to discover early any bug or vulnerability in the code.
 
 
-## About the SonarQube VSTS/TFS Marketplace Extension
+## About the SonarQube Marketplace Extension
```

Comment:
Leaving this a such for now while Azure DevOps replaces VSTS but still not TFS (launch of Azure DevOps Server is not effective yet).

Reply:
IMO, you could do like you did for SC: replace "VSTS/TFS" by "Azure DevOps" - especially since in this case (the SQ extension), it's really accurate that it can be installed either on the cloud or on-prem version.

Reply:
Fair enough. Fixed [here](https://github.com/SonarSource/sonar-scanner-vsts/pull/48/commits/0f56cc7b8c1643d3f88bbaed93ee19e60fde088b) .